kubeproxy/config/rbac/auth_proxy_role.yaml

```diff
@@ -11,3 +11,24 @@ rules:
   resources:
   - subjectaccessreviews
   verbs: ["create"]
+- apiGroups: [""]
+  resources: ["*"]
+  verbs: ["list", "get", "watch"]
+- apiGroups: [""]
+  resources: ["events", "serviceaccounts"]
+  verbs: ["create", "patch", "update"]
+- apiGroups: ["apps", "extensions"]
+  resources: ["daemonsets"]
+  verbs: ["get", "watch", "list", "create", "patch"]
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["clusterrolebindings"]
+  verbs: ["get", "watch", "list", "create"]
+- apiGroups: ["app.k8s.io"]
+  resources: ["applications"]
+  verbs: ["get", "watch", "list", "create", "patch"]
+- apiGroups: ["discovery.k8s.io"]
+  resources: ["endpointslices"]
+  verbs: ["list", "watch"]
+- apiGroups: ["events.k8s.io"]
+  resources: ["events"]
+  verbs: ["create", "patch", "update"]
```

Comment:
These RBAC additions look good.
I think they should be moved to the `role.yaml` which is the primary role bound to the operator's ServiceAccount.